### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by YaNIND** [xSuperr/VanillaBlocks#45]
use bedrockdata https://github.com/supercrafter333/BedrockData can add 1.17 supercrafter333 added 1.17 blocks mapping maybe you can add this

**Comment by YaNIND** [comment]
@edwinyoo44 sorry for tags

**Comment by supercrafter333** [comment]
No, I've tested this in a custom build of Pocketmine, it doesn't Work right.

**Comment by xSuperr** [comment]
Only blocks added by pocketmine bedrock data will be added to comply with poggit rules.
